src/js/messaging/components/buttons/ButtonPrint.jsx

```diff
@@ -23,6 +24,7 @@ class ButtonPrint extends React.Component {
 }
 
 ButtonPrint.propTypes = {
+  onClick: React.PropTypes.func,
 };
 
 export default ButtonPrint;
```

Comment:
`ButtonPrint` actually isn't being used at all right now, but I don't know if a print feature is going to be added in the future.